docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -145,11 +145,18 @@ the displayed RPM value based on the value reported by the module, see the :ref:
 ERPM vs. RPM
 ##############
 As of v4.5.5, ArduCopter flight controllers expect the DSHOT telemetry messages to match the `KISS ESC standard <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_.
-That means that it expects the module to send ERPM/100. Vertiq modules, however, directly send RPM as covered in :ref:`ESC Telemetry <manual_telemetry>`.
+That means that it expects the module to send ERPM/100. Vertiq modules by default, however, directly send RPM as covered in :ref:`ESC Telemetry <manual_telemetry>`.
 
-To convert from ERPM to RPM, the ERPM value should be divided by the pole count of the motor divided by 2. The flight controller is expecting to receive 
-ERPM/100, so when it receives telemetry it tries to calculate RPM using the following formula shown below. This calculation can also be
-seen in the `ArduCopter code for BLHeli telemetry parsing <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_BLHeli/AP_BLHeli.cpp#L1447>`_.
+There are two methods to deal with this. The first, and recommended, option is to configure your module to transmit the ERPM/100 value directly through its telemetry responses. This can be done by changing the value of **Timer Based Protocol RPM Reporting Style** 
+in the Control Center's advanced tab:
+
+.. image:: ../_static/tutorial_images/fc_telemetry_tutorial/erpm_config.png
+    :align: center
+
+.. note:: 
+    This parameter is only available on firmware versions 0.2.0 and up.

```

Comment:
May want to specify **speed** firmware, so that people don't try looking for this on servos or whatever. I know it should be obvious, but people do crazy stuff